reference_guide/intellij_artifacts.md

```diff
@@ -1,100 +1,69 @@
 ---
-title: IntelliJ Platform Artifacts Repository
+title: IntelliJ Platform Artifacts Repositories
 ---
 
-JetBrains maintains a public repository that hosts artifacts related to the IntelliJ Platform, such as binaries and source code. This 
-repository makes artifacts more accessible for plugin authors.
+JetBrains maintains public repositories that host artifacts related to the IntelliJ Platform, such as binaries and source code. These 
+repositories make artifacts more accessible for plugin developers.
 
-The IntelliJ Platform Artifacts Repository contains:
+The IntelliJ Platform artifacts repositories are:
 * [Releases repository](https://www.jetbrains.com/intellij-repository/releases/) for release and EAP versions.
 * [Snapshots repository](https://www.jetbrains.com/intellij-repository/snapshots/) for EAP candidates and the latest EAP for each branch.
+
+Both the Releases and Snapshots repositories have two types of content:
+* Artifacts for cross-platform, ZIP distributions of IntelliJ IDEA binaries and source code.
```

Comment:
The repository now hosts ZIP distributions not only for IntelliJ IDEA, but also for CLion, Rider and MPS, and probably we'll publish other IDEs later. So I suggest to change this to something like "ZIP distributions of some IntelliJ-based IDEs (e.g. IntelliJ IDEA, CLion)".